Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotificationDismissKind.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    public enum InAppNotificationDismissKind
+    {
+        /// <summary>
+        /// When user explicitly dismissed the notification.
+        /// </summary>
+        User = 1,
```

Comment:
Any reason why this is 1,2 rather than 0,1?

Reply:
I copied the behavior of ActivationKind which is a Flag that start from 1. I can revert to the default enum values if necessary.